src/utils/string.ts

```diff
@@ -1,3 +1,16 @@
 export const truncate = (id: string): string => {
   return `${id.slice(0, 16)}...${id.slice(48)}`;
 };
+
+export const toFloat = (value: string, maxDecimals?: number): string => {
+  const result = value.replace(/[^0-9.]/g, '').replace(/(\..*?)\..*/g, '$1');
```

Comment:
Maybe it is worth finding some lib this kind of operations?

Reply:
I'll have a look